lib/screens/other/aboutGuild.dart

```diff
@@ -15,13 +14,16 @@ class AboutGuildPage extends StatelessWidget {
               child: Column(
                 crossAxisAlignment: CrossAxisAlignment.start,
                 children: [
-                  Text(t.aboutGuildGuild, style: Theme.of(context).textTheme.headline6),
+                  Text(t.aboutGuildGuild,
+                      style: Theme.of(context).textTheme.headline6),
                   SizedBox(
                     height: 8,
                   ),
-                  Text("${t.aboutGuildOne}${t.aboutGuildTwo}${t.aboutGuildThree}"
+                  Text(
+                      "${t.aboutGuildOne}${t.aboutGuildTwo}${t.aboutGuildThree}"
                       "\n\n${t.aboutGuildFour}${t.aboutGuildFive}\n\n${t.aboutGuildSix}"
-                      "${t.aboutGuildSeven}${t.aboutGuildEight}\n\n${t.aboutGuildNine}${t.aboutGuildTen}${t.aboutGuildEleven}${t.aboutGuildTwelve}", style: TextStyle(height: 1.5)),
+                      "${t.aboutGuildSeven}${t.aboutGuildEight}\n\n${t.aboutGuildNine}${t.aboutGuildTen}${t.aboutGuildEleven}${t.aboutGuildTwelve}",
```

Comment:
Are there any non-formatting differences in this file? If not, it should probably not be commited. We've not been to stringent on this in the past but it is probably worth trying to avoid this in the future.

Reply:
Ah, yes I made some changes to the file but undid them but probably changed some formatting so it appears like there are changes, forgot to check for that.